### PR TITLE
Logo Designs

### PR DESCRIPTION
**Pull request by jbeguna04**
Hello @ChaitanyaPramod , Greetings, This PR contains:
- Logo Mark
- Logotypes
- SVG and PNG Files
- Updated ReadMe file with the logo

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="https://raw.githubusercontent.com/jbeguna04/findbugs-android/master/LogoDesign/logotype1.png"></p>
```
Comment (ChaitanyaPramod) [inline README.md]: Please use the markdown syntax for image with relative urls
Eg.

    Logo

File: LogoDesign/add [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (ChaitanyaPramod) [inline LogoDesign/add]: This appears to be a superfluous empty file. Please remove it

**Comment by jbeguna04** [comment]
Already updated those things. 
- update ReadMe file using markdown textstyle
- deleted add file

Thank you!

**Comment by ChaitanyaPramod** [comment]
Thank you for the contribution, @jbeguna04

**Pull request merged**